examples/c/sim_erk_crane.c

```diff
@@ -1,25 +1,43 @@
+/*
+ *    This file is part of acados.
+ *
+ *    acados is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation; either
+ *    version 3 of the License, or (at your option) any later version.
+ *
+ *    acados is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Lesser General Public
+ *    License along with acados; if not, write to the Free Software Foundation,
+ *    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+// external
 #include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
-
-#include "acados/sim/sim_common.h"
-#include "acados/sim/sim_erk_integrator.h"
-#include "acados/sim/sim_casadi_wrapper.h"
-
-#include "acados/utils/print.h"
-#include "acados/utils/timing.h"
-#include "acados/utils/types.h"
-#include "acados/utils/create.h"
+// acados
+#include <acados_c/sim.h>
+#include <acados_c/options.h>
+// NOTE(nielsvd): required to cast memory etc. should go.
+#include <acados/sim/sim_common.h>
+#include <acados/sim/sim_erk_integrator.h>
+#include <acados/sim/sim_casadi_wrapper.h>
 
 #include "examples/c/crane_model/crane_model.h"
 
 // blasfeo
-#include "external/blasfeo/include/blasfeo_target.h"
-#include "external/blasfeo/include/blasfeo_common.h"
-#include "external/blasfeo/include/blasfeo_d_aux.h"
-#include "external/blasfeo/include/blasfeo_d_aux_ext_dep.h"
-#include "external/blasfeo/include/blasfeo_v_aux_ext_dep.h"
-#include "external/blasfeo/include/blasfeo_d_blas.h"
+#include <blasfeo/include/blasfeo_target.h>
```

Comment:
I did this because in principle the interface is a separate library from acore, blasfeo, hpipm, etc. We can change to ``#include "..."`` if preferred.

Reply:
I don't have a strong opinion, as long as it is consistent (through each library at least). @roversch?